src/main.py

```diff
@@ -115,6 +115,12 @@ def methodology(lang, year):
     return render_template('%s/%s/methodology.html' % (lang, year))
 
 
+@app.route('/<lang>/<year>/accessibility-statement')
```

Comment:
Is it possible to route this in a way that only varies by language and not year? Does it make sense for this statement to be site-wide as opposed to year-specific?

Reply:
Ideally, but this is complicated as it means we'd lose the styling, navigation...etc.

I think ignore this for now and revisit when we look at 2020 and how that will interact with 2019 site - will it have the same styling? Different one? Will we have more generic pages?

Alternatively we could host this statement in our GitHub repository like we do our licence instead of directly on our site?

Reply:
Maybe change the route to exclude year and hardcode 2019 in the `render_template` call for now?

```suggestion
@app.route('/<lang>/accessibility-statement')
```

Reply:
Fixed in https://github.com/HTTPArchive/almanac.httparchive.org/pull/665/commits/b2cbdab0a95ef731ecfe9781d842c10263e6145c